src/lib/notifications.ts

```
@@ -0,0 +1,60 @@
+
+Notifications.setNotificationHandler({
+  handleNotification: async () => ({
+    shouldPlaySound: false,
+    shouldSetBadge: false,
+    shouldShowBanner: true,
+    shouldShowList: true,
+  }),
+});
+
+export async function registerForPushNotificationsAsync() {
+  let token;
```

ESLint

[error] Delete `;`

(prettier/prettier)